src/couch/src/couch_os_process.erl

```diff
@@ -92,7 +92,7 @@ readjson(OsProc) when is_record(OsProc, os_proc) ->
             case ?JSON_DECODE(Line) of
                 [<<"log">>, Msg] when is_binary(Msg) ->
                     % we got a message to log. Log it and continue
-                    couch_log:info(
+                    couch_log:error(
```

Comment:
I'm curious why you increased the log level here.

Reply:
Oh good find. That's a debugging remnant. I'll revert it.

Reply:
Updated with a fixup commit to revert the change.